<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>downloadClips</class>
 <widget class="QMainWindow" name="downloadClips">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>816</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>29</green>
        <blue>29</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>36</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>14</green>
        <blue>14</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>19</red>
        <green>19</green>
        <blue>19</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>146</red>
        <green>151</green>
        <blue>152</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>29</green>
        <blue>29</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>120</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>14</green>
        <blue>14</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>36</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>14</green>
        <blue>14</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>19</red>
        <green>19</green>
        <blue>19</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>146</red>
        <green>151</green>
        <blue>152</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>29</green>
        <blue>29</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>38</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>14</green>
        <blue>14</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>14</green>
        <blue>14</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>216</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>227</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>14</green>
        <blue>14</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>19</red>
        <green>19</green>
        <blue>19</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>14</green>
        <blue>14</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>253</red>
        <green>2</green>
        <blue>2</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>244</red>
        <green>77</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>227</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>221</red>
        <green>12</green>
        <blue>12</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>120</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>29</green>
        <blue>29</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Clip Download Options</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>770</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="addingToDBLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>740</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Downloading Clips</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="downloadLog">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>561</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QComboBox" name="gameSelect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="autoDownloadInfo">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>160</y>
      <width>231</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addGame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to autodownloader</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopFinding">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>670</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Finding</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startFinding">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Finding</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Downloader Games</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearGames">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear autodownloader list</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="clipBinInformation">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>551</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip Bin Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="amountCurrentPass">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>697</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount in current pass:</string>
    </property>
   </widget>
   <widget class="QLabel" name="downloadProgressAmount">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>720</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Download progress:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentDownloadGame">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>740</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Game:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshGameClips">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↻</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startDownloading">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>710</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Downloading</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopDownloading">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>740</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Downloading</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startAuto">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopAuto">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Auto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNewGame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Game</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bulkDownloadAmount">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>710</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>690</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bulk Download Amount</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateStatus">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Missing Clip Status</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>872</x>
      <y>0</y>
      <width>21</width>
      <height>831</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Update's the clip bin database by checking if the file exists. If it doesn't the database will mark the clip as missing.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="gameSelectToDelete">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>230</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteClips">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Clips</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>160</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete's all downloaded clips for selected game. The clips will be available to redownload if the game is added to the autodownloader.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="accountInfo">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>641</y>
      <width>256</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>600</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Accounts</string>
    </property>
   </widget>
   <widget class="QComboBox" name="userToRemove">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>510</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>480</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeUser">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>540</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNewUser">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>400</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New User</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>340</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>370</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>310</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>580</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(Restart the server to update any changes to accounts)</string>
    </property>
   </widget>
   <widget class="QLabel" name="userAddStatus">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>430</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="finishVidDirectory">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Finished Video Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clipBinDirectory">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Clip Bin</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
